### PR TITLE
Rewrite voor de nieuwe public-travel-information api & python3.0

### PR DESCRIPTION
**Pull request by Squixx**
Gezien ik geen login kan krijgen voor de oude api, een update voor toegang tot de nieuwe.

includes: dev docker

Werkt op basis van Ocp-Apim-Subscription-Key

Ik dacht ik doe t via een update van deze package ipv wat nieuws schrijven,

Doel is uiteindelijk dit in Homeassistant beschikbaar te maken ;)

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Do you still have plans to continue on this? Because it is now October, the old API is completely deprecated and no longer useful. Would be nice if the integration could soon be used again in Home Assistant.

**Comment by aquatix** [comment]
I've been more busy than expected, but will be reviewing later today. I fully expect it to be merged after that, and I'll release a new version after that.

**Comment by klaasnicolaas** [comment]
Perfect. From Home Assistant we have now placed [an alert](https://alerts.home-assistant.io/#nederlandse_spoorwegen.markdown), so that everyone knows that the integration is not working. If it does work again I would like to hear it, so we can remove the alert 😉

File: setup.py [hunk 1]
```diff
@@ -1,60 +1,60 @@
-"""
-A setuptools based setup module.
```
Comment (aquatix) [inline setup.py]: Have the line-endings of all files changed? I would really prefer it if you could re-do the PR without these invasive changes, as diffing is quite hard, and revision history is lost.

**Comment by jorneilander** [comment]
@Squixx @aquatix
Any update on this issue? I was hoping to use the HA sensor to warn me on train disruptions in the early morning :P

**Comment by metbril** [comment]
I'd like to use the integration again, too.

**Comment by aquatix** [comment]
Thanks everyone on your patience with this. I'm currently working on PR based on the above and will be committing shortly.

**Comment by Squixx** [comment]
closed as already implemented ;)

**Pull request closed without merging**